vocode/streaming/vector_db/pinecone.py

```diff
@@ -1,32 +1,119 @@
 import asyncio
+import logging
```

Comment:
let's add a comment somewhere here that this is the same as the langchain implementation but the calls to pinecone are async

Reply:
i.e.

```
Adapted from: path/to/langchain
```